.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -22,5 +22,5 @@ If applicable, add screenshots or logs to help explain your problem.
 
 **Environment (please complete the following information):**
 - OS: [e.g. Debian GNU/Linux]
-- Meilisearch version: [e.g. v.0.20.0]
+- Meilisearch version: [e.g. v.0.29.1]
```

Comment:
Do you sure you want to update the `bug_report` template on the same PR?

Reply:
I came by that mistake and thought I should fixed it. But if you prefer I can open another PR with it

Reply:
My bad, you are right it is not needed as an update. Its just an example! Though I'm updating it for rust as it seems less out-dated.

Reply:
Like you prefer just to be sure